code/modules/power/gravitygenerator.dm

```diff
@@ -453,6 +458,9 @@ GLOBAL_LIST_EMPTY(gravity_generators)
 	for(var/obj/machinery/gravity_generator/part as anything in generator_parts)
 		SET_PLANE(part, PLANE_TO_TRUE(part.plane), new_turf)
 
+/obj/machinery/gravity_generator/main/proc/get_radioactive_nebula_shielding()
+	return on ? radioactive_nebula_shielding : 0
```

Comment:
```suggestion
/obj/machinery/gravity_generator/main/proc/get_radioactive_nebula_shielding()
	SIGNAL_HANDLER
	return on ? radioactive_nebula_shielding : 0
```
Likewise, signal procs should never return anything but a bitflag, it's a bit of a misuse of signals. Might need another way to do this